channel.go

```diff
@@ -259,11 +259,16 @@ func (ch *Channel) UnBanUser(targetID string, options map[string]string) error {
 
 // CreateChannel creates new channel of given type and id or returns already created one
 func (c *Client) CreateChannel(chanType string, chanID string, userID string, data map[string]interface{}) (*Channel, error) {
+	var membersPresent bool
+	if len(data) > 0 {
+		_, membersPresent = data["members"]
```

Comment:
this line is enough because nil map is accessible by default value

https://play.golang.org/p/HykDxMpGcUe

Reply:
updated at #13